misc/jhbuildrc-custom

```diff
@@ -6,7 +6,7 @@ import os
 checkoutroot = os.path.expanduser("~/jhbuild_checkoutroot")
 prefix = os.path.expanduser("~/jhbuild_prefix")
 modulesets_dir = os.environ["QL_OSXBUNDLE_MODULESETS_DIR"]
-moduleset = ["bootstrap", "quodlibet"]
```

Comment:
Why do you remove the bootstrap moduleset? Does jhbuild still find bootstrap.modules?